### PR TITLE
Cut v2.37.0-rc.0

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@o11y.eu>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]
:taco:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+Following data loss by users due to lack of unified buffer cache in OpenBSD, we
+will no longer release Prometheus upstream for OpenBSD until a proper solution is
+found.
+
+* [FEATURE] Nomad SD: New service discovery for Nomad built-in service discovery. #10915
+* [ENHANCEMENT] Kubernetes SD: Allow attaching node labels for endpoint role. #10759
+* [ENHANCEMENT] PromQL: Optimise creation of signature with/without labels. #10667
+* [ENHANCEMENT] TSDB: Memory optimizations. #10873 #10874
+* [ENHANCEMENT] TSDB: Reduce sleep time when reading WAL. #10859 #10878
+* [BUGFIX] Alerting: Fix alertmanager targets not being picked when alerts where queued. #10948
```
Comment (juliusv) [inline CHANGELOG.md:14]:
```suggestion
* [BUGFIX] Alerting: Fix Alertmanager targets not being uploaded when alerts were queued. #10948
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+* [BUGFIX] Hetzner SD: Make authentication files relative to Prometheus config file. #10813
+* [BUGFIX] Promtool: Fix promtool check config not erroring properly on failures. #10952
```
Comment (juliusv) [inline CHANGELOG.md:16]:
```suggestion
* [BUGFIX] Promtool: Fix `promtool check config` not erroring properly on failures. #10952
```

**Review by bwplotka — APPROVED** [review]
👍🏽

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,32 @@
 # Changelog
 
+## 2.37.0-rc.0 / 2022-07-04
+
+Following data loss by users due to lack of unified buffer cache in OpenBSD, we
```
Comment (bwplotka) [inline CHANGELOG.md:5]: Should we add issue for reference?

**Pull request merged**